code/datums/brain_damage/mild.dm

```diff
@@ -264,3 +264,36 @@
 			speak_dejavu += speech_args[SPEECH_MESSAGE]
 	else
 		speak_dejavu += speech_args[SPEECH_MESSAGE]
+
+/datum/brain_trauma/mild/redacted
+	name = "Confidentiality Trauma"
+	desc = "Patient's language neurons seem to be warped in a strange manner, resulting in them being unable to speak properly."
+	scan_desc = "<b>\[REDACTED]</b>"
+	gain_text = "<span class='warning'>You feel the need to <b>\[REDACTED]</b></span>"
+	lose_text = "<span class='notice'>You no longer feel the need to <b>\[REDACTED]</b>.</span>"
+
+/datum/brain_trauma/mild/redacted/handle_speech(datum/source, list/speech_args)
+	var/message = speech_args[SPEECH_MESSAGE]
+	if(message)
+		var/list/message_split = splittext(message, " ")
+		var/list/new_message = list()
+
+		for(var/word in message_split)
+			var/suffix = ""
+			var/suffix_foundon = 0
+			for(var/potential_suffix in list("." , "," , ";" , "!" , ":" , "?"))
+				suffix_foundon = findtext(word, potential_suffix, -length(potential_suffix))
+				if(suffix_foundon)
+					suffix = potential_suffix
+					break
+
+			if(suffix_foundon)
+				word = copytext(word, 1, suffix_foundon)
+
+			word = html_decode(word)
+
+			if(prob(25))
+				word = pick(list("<b>\[REDACTED]</b>", "<b>\[CLASSIFIED]</b>"))
```

Comment:
```suggestion
				word = pick(list("<b>\[REDACTED]</b>", "<b>\[CLASSIFIED]</b>", "<b>\[EXPLETIVES REDACTED]</b>", "<b>\[EXPUNGED]</b>"))
```
lol two isn't funny enough

also are you sure the bold tags work in chat or should you be using +[REDACTED]+ for the text parsing instead

Reply:
the bold tags work in chat, i've given the trauma to myself and tested it

Reply:
EXPLETIVES REDACTED seems weird but i'll add EXPUNGED